components/gitpod-db/src/typeorm/user-db-impl.ts

```diff
@@ -419,7 +419,7 @@ export class TypeORMUserDBImpl implements UserDB {
         } else {
             var user: MaybeUser;
             if (accessToken.user) {
-                user = await this.findUserById(accessToken.user.id)
+                user = await this.findUserById(String(accessToken.user.id))
```

Comment:
nit: we never use `String`. In this case maybe `.toString()` would be a more idiomatic solution.